components/icons/DocsArrow.js

```diff
@@ -0,0 +1,7 @@
+export default function DocsArrow({ isDropDown, activeDropDown, onClick, className }) {
```

Comment:
for better understanding of a newcommer we should have `activeDropDownItem` instead of `activeDropDown`